app/letsencrypt_service

```diff
@@ -8,6 +8,8 @@ ACME_CA_URI="${ACME_CA_URI:-https://acme-v01.api.letsencrypt.org/directory}"
 DEFAULT_KEY_SIZE=4096
 REUSE_ACCOUNT_KEYS="$(lc ${REUSE_ACCOUNT_KEYS:-true})"
 REUSE_PRIVATE_KEYS="$(lc ${REUSE_PRIVATE_KEYS:-false})"
+MAX_VALIDITY=7776000
```

Comment:
Maybe rename this variable to `MIN_VALIDITY_CAP` (or something else along the same line) to avoid possible confusion with the default duration of certs generated by LE / the ACME CA used ?

Reply:
done